package.json

```diff
@@ -186,5 +186,8 @@
       "stories/**/*.js"
     ],
     "cache": false
+  },
+  "optionalDependencies": {
+    "fsevents": "^2.0.1"
```

Comment:
Oh! Was this due to Node 10? (Or something)
I experienced this randomly on one of my personal machines.

Reply:
TBH I'm not sure what the root cause is, I'm running 11, I had storybook running (and hsapp) and then the laptop's fans went crazy after a little while even when I wasn't doing anything, peaked at the activity monitor and both node processes were above 100%, after a little search, tried the `fsevents` fix and haven't have the issue again.